packages/surveys/src/utils/results-availability/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./has-user-responded";
+export * from "./migrate-form-properties-settings";
+export * from "./should-display-results";
```

Comment:
as I suggested in #384 - my preference would be to move these fns into this file rather than re-export them.

Reply:
that's fine, but having them individual makes them stub-able in tests